index.html

```diff
@@ -1,46 +1,124 @@
+        }
+
+        .read-more:hover {
+            text-decoration: underline;
+        }
+
+        footer {
+            text-align: center;
+            padding: 20px;
+            background-color: lightgray;
+            color: dimgray;
+            position: relative;
+            bottom: 0;
+            width: 100%;
+            margin-top: 20px;
+        }
     </style>
-  </head>
-  <body>
-    <header><h1>🧐 CYF Coursework Disposable Branch Previews</h1></header>
+</head>
+<body>
+    <header>
+        <h1>Web Development Basics</h1>
+        <p>Understanding key concepts in web development</p>
+    </header>
+
     <main>
-      <ol>
-        <li>
-          <h2><a href="/Wireframe">Project 1: Wireframe</a></h2>
-          <p>
-            Mentors:
-            <a href="Wireframe/readme.md">open the assignment in a tab</a>
-          </p>
-        </li>
-        <li>
-          <h2><a href="/Form-Controls">Project 2: Form Controls</a></h2>
-          <p>
-            Mentors:
-            <a href="Form-Controls/readme.md">open the assignment in a tab</a>
-          </p>
-        </li>
-      </ol>
+        <section class="articles">
+            <article class="article-card">
+                <h2>Purpose of a README File</h2>
+                <p>A README file is a document, typically written in plain text or Markdown, that provides an overview of a project. Its purpose is to give users and contributors essential information such as the project's description, installation instructions, usage guidelines, and contribution rules. It serves as the first point of reference for anyone interacting with the project, ensuring clarity and ease of use.</p>
+                <a href="#" class="read-more">Read More</a>
+            </article>
+
+            <article class="article-card">
+                <h2>Purpose of a Wireframe</h2>
+                <p>A wireframe is a visual blueprint or skeletal outline of a webpage or application. Its purpose is to plan the layout and structure before detailed design and development begin. It helps stakeholders visualize the user interface and user experience, focusing on functionality and content placement without distractions from colors or graphics.</p>
+                <a href="#" class="read-more">Read More</a>
+            </article>
+
+            <article class="article-card">
+                <h2>What is a Branch in Git?</h2>
+                <p>A branch in Git is a lightweight movable pointer to a commit. Its purpose is to allow multiple lines of development to occur simultaneously. Developers use branches to work on features, fixes, or experiments independently from the main codebase, merging changes back when ready, which helps manage collaboration and version control efficiently.</p>
+                <a href="#" class="read-more">Read More</a>
+            </article>
+        </section>
     </main>
-    <footer><a href="HOW_TO_REVIEW.md">HOW TO REVIEW MD</a></footer>
-  </body>
+
+    <footer>
```

Comment:
The acceptance criteria asks for this footer to be fixed to the bottom of the viewport. Could you update this to it's position is fixed to the bottom of the viewport?